heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerMain.java

```diff
@@ -378,7 +377,7 @@ public void manageTopology()
       // TODO(mfu): timeout should read from config
       SchedulerStateManagerAdaptor adaptor = new SchedulerStateManagerAdaptor(statemgr, 5000);
 
-      validateRuntimeManage(adaptor, topologyName);
+      boolean foundData = validateRuntimeManage(adaptor, topologyName);
```

Comment:
foundData - does not sound like a good name. Any other choices?

Reply:
sigh.

Maosong suggested "exist" but I feel it is not good either. :(

This is a trick case and naming is tricky too.

Reply:
this data is all about executionState - is it correct? if so we can name it as
```
executionStateExists
```
will do?

Reply:
- dataFound
- topologyFound
- topologyExist
or reverse them:
- topologyNotFound
- topologyNotExist

Which one sounds better?

Reply:
it is not only about executionState.

We got a report later that executionStateExists is available but definition is not. Basically none of them is safe.

This flag is from this logic:
Boolean isTopologyRunning = adaptor.isTopologyRunning(topologyName);
boolean found = isTopologyRunning != null && isTopologyRunning.equals(Boolean.TRUE);

That is why I used isRunning at beginning, which is not ideal for sure.

Another option is to skip the validation altogether for kill command and no need to worry about which data is available. The checks and flags here are used to show different message, except envronment check if execution state is available (which isn't critical for this command to me).